src/home/Benefits.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './Benefits.css';
+import BenefitsContainer from './BenefitsContainer'
+import imgPartner from "../assets/images/benefit_partner.png"
+import imgMember from "../assets/images/benefit_member.png"
+import imgMentor from "../assets/images/benefit_mentor.png"
+
+export default function Benefits(){
+    return(
+        <div>
+            <div className="mission">
+                <span style={{color:"#EF9425"}}>Our mission </span> is to provide a platform where <br/>everyone can reach their full potential - <span style={{fontWeight:250,fontStyle:"italic"}}>starting with you</span> 
+            </div>
+            <BenefitsContainer img={imgPartner} heading={<span>Become a <span style={{color:"#EF9425"}}>partner</span></span>} itemClass='benefitContainer' buttonText='Become a partner'>
+                        Join our cause by signing up as a partner and help us aquire the resources we need for our members 
```

Comment:
Hey @DhaneshShetty, please provide proper indentations in Benefits.jsx and BenefitsContainer.jsx, it would not affect any feature but will help in maintaining clean code. 😉